### ISSUE TITLE
ng commands not recognized on MacOS 10.13.3 with Angular 5.2.0

### ISSUE THREAD

**Issue by LargoLobo** [angular/angular-cli#9844]
npm install globally required: ~/.npm-global at root.
  npm install -g @angular/cli  didn't work unless I was in this new global directory at .npm-global/bin/ng

Once angular installed, the ng command not recognized. I ran "npm install" inside my project directory to load dependencies and then "npm start" which opened the angular server as expected.  In the package.json there is a script called "ng" , but it doesn't do anything. If I add any property like "npm ng doc" , I get an error.  The "npm test" script worked, but after the page opened generated an MacOS error on the console.log.

Basically, the angular-cli isn't working with the Mac...?

**Comment by clydin** [comment]
Hello, we reviewed this issue and determined that it doesn't fall into the bug report or feature request category. This issue tracker is not suitable for support requests, please repost your issue on [StackOverflow](http://stackoverflow.com/) using tag `angular-cli`.

If you are wondering why we don't resolve support issues via the issue tracker, please [check out this explanation](https://github.com/angular/angular-cli/blob/master/CONTRIBUTING.md#-got-a-question-or-problem).
